src/pages/about-us.js

```diff
@@ -1,11 +1,12 @@
 import Head from 'components/Head'
 import aboutUsData from '../../data/about-us'
-import { AboutUsHero, OurStory, Investors } from 'blocks'
+import { AboutUsHero, OurStory, Investors, JobOffers } from 'blocks'
```

Comment:
I took a look at the general architecture of the codebase and I really like the architecture of it! Super logical